eventmesh-common/src/test/java/org/apache/eventmesh/common/config/ConfigurationWrapperTest.java

```diff
@@ -31,7 +31,7 @@ public class ConfigurationWrapperTest {
     public void before() {
         String file = ConfigurationWrapperTest.class.getResource("/configuration.properties").getFile();
```

Comment:
this config file doesn't exist right ?

Reply:
This is not my code

Reply:
I get it. I changed the code here, which made CI fail all the time